internal/operator/pgdump/restore.go

```diff
@@ -75,7 +76,8 @@ func Restore(namespace string, clientset kubeapi.Interface, task *crv1.Pgtask) {
 	taskName := task.Name
 
 	jobFields := restorejobTemplateFields{
-		JobName:             "pgrestore-" + task.Spec.Parameters[config.LABEL_PGRESTORE_FROM_CLUSTER] + "-from-" + fromPvcName + "-" + util.RandStringBytesRmndr(4),
+		JobName: fmt.Sprintf("pgrestore-%s-%s", task.Spec.Parameters[config.LABEL_PGRESTORE_FROM_CLUSTER],
+			util.RandStringBytesRmndr(4)),
```

Comment:
fwiw, [`k8s.io/apimachinery/pkg/util/rand.String()`](https://pkg.go.dev/k8s.io/apimachinery@v0.17.4/pkg/util/rand?tab=doc#String) generates similar random strings. I believe it's how Pods get their names: https://github.com/kubernetes/apiserver/blob/v0.19.0/pkg/storage/names/generate.go#L53